pkgs/development/libraries/gstreamer/default.nix

```diff
@@ -3,69 +3,48 @@
   lib,
   stdenv,
   callPackage,
-  AVFoundation,
-  AudioToolbox,
-  Cocoa,
-  CoreFoundation,
-  CoreMedia,
-  CoreServices,
-  CoreVideo,
-  DiskArbitration,
-  Foundation,
-  IOKit,
-  MediaToolbox,
-  OpenGL,
-  Security,
-  SystemConfiguration,
-  VideoToolbox,
-  xpc,
   ipu6ep-camera-hal,
   ipu6epmtl-camera-hal,
+  apple-sdk_13,
 }:
 
+let
+  callGstreamerPackage =
+    path: attrs:
+    (callPackage path attrs).overrideAttrs (oldAttrs: {
+      buildInputs = oldAttrs.buildInputs ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_13 ];
+    });
```

Comment:
Let’s avoid `overrideAttrs` and spooky action at a distance – we can do this like Qt 6 by including `apple-sdk_gstreamer = apple-sdk_13;` in the scope and listing it in the `buildInputs` of each sub‐package. A little noisy, but explicit, and crucially centralizes the SDK version decision.

Reply:
I don't like the noise, but I will oblige. :)

Reply:
It’s at least a big net reduction in noise!